hoard/sources/whoas.py

```diff
@@ -129,7 +129,9 @@ def create_from_whoas_dim_xml(data: str) -> Dataset:
         ):
             kwargs["productionPlace"] = field.text
         if field.attrib["element"] == "relation" and "qualifier" not in field.attrib:
-            publications.append(Publication(publicationCitation=field.text))
+            publications.append(
```

Comment:
Our WHOAS metadata mapping does not include the publicationID or publicationIDType fields, and we should not arbitrarily add an ID type of URL. See https://mitlibraries.atlassian.net/browse/RDRP-354 for details on how we should handle this issue. I would remove this change and do that in a separate PR.